cipher_suite_go114_test.go

```diff
@@ -0,0 +1,49 @@
+// +build go1.14
+
+package dtls
+
+import (
+	"testing"
+)
+
+func TestInsecureCipherSuites(t *testing.T) {
+	r := InsecureCipherSuites()
+
+	if len(r) != 0 {
+		t.Fatalf("Expected no insecure ciphersuites, got %d", len(r))
+	}
+}
+
+func TestCipherSuites(t *testing.T) {
+	ours := allCipherSuites()
+	theirs := CipherSuites()
+
+	if len(ours) != len(theirs) {
+		t.Fatalf("Expected %d CipherSuites, got %d", len(ours), len(theirs))
+	}
+
+	for i, s := range ours {
+		t.Run(s.String(), func(t *testing.T) {
+			c := theirs[i]
```

Comment:
golangci-lint on Go1.14 seems saying:
```
$ golangci-lint run
cipher_suite_go114.go:23:2: Consider preallocating `res` (prealloc)
	var res []*tls.CipherSuite
	^
cipher_suite_go114_test.go:27:16: Using the variable on range scope `i` in function literal (scopelint)
			c := theirs[i]
			            ^
cipher_suite_go114_test.go:28:22: Using the variable on range scope `s` in function literal (scopelint)
			if c.ID != uint16(s.ID()) {
			                  ^
cipher_suite_go114_test.go:29:49: Using the variable on range scope `s` in function literal (scopelint)
				t.Fatalf("Expected ID: 0x%04X, got 0x%04X", s.ID(), c.ID)
				                                            ^
cipher_suite_go114_test.go:32:17: Using the variable on range scope `s` in function literal (scopelint)
			if c.Name != s.String() {
```
scopelint errors are not actual problem on non Parallel tests.

Reply:
I believe I've fixed them all, for good measure.